.env.sample

```
@@ -1,3 +1,7 @@
+# Contentful API keys
+CONTENTFUL_SPACE_ID=
+CONTENTFUL_ACCESS_TOKEN=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The CONTENTFUL_ACCESS_TOKEN key should go before the CONTENTFUL_SPACE_ID key

(UnorderedKey)